tritium-core/src/main/java/com/palantir/tritium/event/CompositeInvocationEventHandler.java

```diff
@@ -51,6 +51,16 @@ private CompositeInvocationEventHandler(List<InvocationEventHandler<InvocationCo
         }
     }
 
+    @Override
+    public boolean asyncSupport() {
+        for (InvocationEventHandler<?> handler : handlers) {
+            if (!handler.asyncSupport()) {
+                return false;
+            }
+        }
+        return true;
```

Comment:
Tritium bundles all handlers together using the composite handler, I think this will disable async support if the tracing handler is enabled.

Reply:
I not sure CompositeInvocationEventHandler is the right abstraction for InvocationEventProxy any longer given this change.